src/test/java/com/cook4me/controller/OfferManagerTest.java

```diff
@@ -0,0 +1,78 @@
+package com.cook4me.controller;
+
+import com.cook4me.model.Offer;
+import org.junit.*;
+import static org.junit.Assert.*;
+
+
+/**
+ * Created by Gaba on 2016-12-08.
+ */
+
+public class OfferManagerTest {
+    private Offer offer;
+    private OfferManager offerManager;
+    private Offer nullOffer;
+    private Offer almostNullOffer;
+
+    @Before
+    public void setUp(){
+        offerManager = new OfferManager();
+        offer = new Offer("bigos", "Krakow", "20.12.2016", "yummy stuff", "880880880");
+        nullOffer = null;
+        almostNullOffer = new Offer(null, null, null, null, null);
+
+    }
+
+    @Test
+    public void testStoreOffer(){
+        offerManager.store(offer);
+
+        assertEquals(1, offerManager.getOffers().size());
+        assertEquals(offer, offerManager.getOffer(offer.getId()));
+        assertEquals("bigos", offerManager.getOffer(offer.getId()).getName());
+        assertEquals(offer, offerManager.getOffers().get(0));
+        assertNull(offerManager.getOffer(offer.getId() + 24));
+
+    }
+    @Test
+    public void testStoreOffer2(){
+        offerManager.store(offer);
+        offerManager.store(offer);
+
+        assertEquals(2, offerManager.getOffers().size());
+
+    }
+
+    @Test
+    public void testStoreOffer3(){
```

Comment:
this and previous test have the same functionatily, what do you think about parameterization?